packages/generator/src/edmx-to-vdm/v4/operation.ts

```diff
@@ -20,95 +29,280 @@ const logger = createLogger({
 
 const extractResponse = (response: string) => `${response}.value`;
 
-interface JoinedOperationData {
-  operationImport: EdmxOperationImport;
-  operation: EdmxOperation;
+function splitMissingOperation(
+  operationImports: EdmxOperationImport[],
+  operations: EdmxOperation[]
+): [EdmxOperationImport[], EdmxJoinedOperation[]] {
+  return operationImports.reduce<
+    [EdmxOperationImport[], EdmxJoinedOperation[]]
+  >(
+    ([withoutOperation, withOperation], curr) => {
+      const operation = findOperationByImportName(
+        operations,
+        curr.operationName
+      );
+      if (operation) {
+        withOperation.push({
+          Name: curr.Name,
+          operationName: curr.operationName,
+          Parameter: operation.Parameter,
+          ReturnType: operation.ReturnType,
+          IsBound: operation.IsBound.toLowerCase() === 'true',
+          operationType: curr.operationType
+        });
+      } else {
+        withoutOperation.push(curr);
+      }
+      return [withoutOperation, withOperation];
+    },
+    [[], []]
+  );
+}
+
+function extractEntitySetName(type: string): string {
+  const components = type.split('.');
+  return components.pop()!;
+}
+
+function splitMissingParameter( // fixme name
+  operations: EdmxJoinedOperation[]
+): [EdmxJoinedOperation[], EdmxJoinedOperation[]] {
+  return operations.reduce<[EdmxJoinedOperation[], EdmxJoinedOperation[]]>(
+    ([validOperations, withoutParameter], curr) => {
+      if (!curr.IsBound) {
+        return [[...validOperations, curr], withoutParameter];
+      }
+
+      if (!curr.Parameter.length) {
+        return [validOperations, [...withoutParameter, curr]];
+      }
+
+      const entitySetName = extractEntitySetName(curr.Parameter[0].Type);
+      if (entitySetName) {
+        const bound = {
+          ...curr,
+          entitySetName,
```

Comment:
[req] Also put here something like `edmxEntityName` or so to indicate that it is the raw name with potential `A_`